deploy/helm/sumologic/values.yaml

```diff
@@ -82,6 +82,36 @@ sumologic:
         helm.sh/hook-weight: "0"
         helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
 
+  ## Configuration of http sources
+  ## name: source name
+  ## value: source values - it's visible in sumologic platform
+  ## endpoint: binds source with fluentd configuration: (fluentd.metrics.output[])
+  sources:
```

Comment:
This would better be a default sources + additional sources. Or something like that.